xyz.safeworlds.hiit.yml

```diff
@@ -0,0 +1,40 @@
+id: xyz.safeworlds.hiit
+runtime: org.gnome.Platform
+runtime-version: "45"
+sdk: org.gnome.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable # 23.08
+- org.freedesktop.Sdk.Extension.llvm16
+command: hiit
+finish-args:
+- --share=ipc
+- --socket=fallback-x11
+- --socket=wayland
+- --socket=pulseaudio
+- --device=dri
+- --env=RUST_LOG=gtk_rust_template=debug
```

Comment:
why would you run the app with the verbose debug output?

Reply:
Exactly. Removed logging related env vars.